com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/jobs/CleanUpJob.java

```diff
@@ -114,12 +113,12 @@ private Date getExpirationTime() {
         return cleanUpTimestamp;
     }
 
-    private void abortOldOperationsInStateError(Date expirationTime) {
+    private void abortOldOperationsInActiveState(Date expirationTime) {
         LOGGER.info("Aborting operations started before: " + expirationTime.toString());
-        List<Operation> activeOperationsInStateError = getActiveOperationsInStateError(expirationTime);
-        List<String> operationsInErrorIds = getProcessIds(activeOperationsInStateError);
-        executeAbortOperationAction(operationsInErrorIds);
-        LOGGER.info("Aborted operations count : " + operationsInErrorIds.size());
+        List<Operation> activeOperations = getOperationsInActiveState(expirationTime);
+        List<String> activeOperationsIds = getProcessIds(activeOperations);
+        executeAbortOperationAction(activeOperationsIds);

```

Comment:
As we are in the fixing mode, could you add some error handling here? :)
The executeAbortOperationAction could throw various errors related with the Activiti.
Maybe we should catch them here and decide whether to ignore them ? For instance some logging?

Reply:
tryToAbortOperation will catch any exception produced by the Activiti call and log an error but continue the execution for the next operations in the stream.